### PR TITLE
[Sprint 8] Feature Notification Device Continue

### PR DESCRIPTION
**Pull request by tuanlvt-1117**

### CODE REVIEW

File: app/src/main/java/com/framgia/forder/data/source/local/MyFirebaseMessagingService.java [hunk 1]
```diff
@@ -18,30 +18,31 @@
...
-        PendingIntent pendingIntent =
-                PendingIntent.getActivity(this, 0, intent, PendingIntent.FLAG_ONE_SHOT);
+        PendingIntent pendingIntent = PendingIntent.getActivity(this, 0 /* Request code */, intent,
+                PendingIntent.FLAG_ONE_SHOT);
 
         Uri defaultSoundUri = RingtoneManager.getDefaultUri(RingtoneManager.TYPE_NOTIFICATION);
         NotificationCompat.Builder notificationBuilder =
-                new NotificationCompat.Builder(this).setSmallIcon(R.mipmap.ic_launcher)
-                        .setContentTitle(messageTitle)
+                new NotificationCompat.Builder(this).setSmallIcon(R.drawable.ic_launcher)
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/data/source/local/MyFirebaseMessagingService.java]: WARNING: [IconColors] Notification icons must be entirely white
 Notification icons and Action Bar icons should only white and shades of gray. See the Android Design Guide for more details. Note that the way Lint decides whether an icon is an action bar icon or a notification icon is based on the filename prefix: `ic_menu_` for action bar icons, `ic_stat_` for notification icons etc. These correspond to the naming conventions documented in http://developer.android.com/guide/practices/ui_guidelines/icon_design.html

File: app/src/main/java/com/framgia/forder/data/source/local/MyFirebaseInstanceIDService.java [hunk 2]
```diff
@@ -10,12 +10,12 @@
...
+     private static final String TAG = "MyFirebaseInstanceIDService";
 
     @Override
     public void onTokenRefresh() {
-
-        String refreshedToken = FirebaseInstanceId.getInstance().getToken();
-        Log.d(TAG, refreshedToken);
+        super.onTokenRefresh();
+        String token = FirebaseInstanceId.getInstance().getToken();
+        Log.d(TAG, "onTokenRefresh: " + token);
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/data/source/local/MyFirebaseInstanceIDService.java]: ERROR: [LongLogTag] The logging tag can be at most 23 characters, was 27 (MyFirebaseInstanceIDService)
 Log tags are only allowed to be at most 23 tag characters long.

File: app/src/main/java/com/framgia/forder/data/source/local/MyFirebaseInstanceIDService.java [hunk 3]
```diff
@@ -10,12 +10,12 @@
...
+    private static final String TAG = "MyFirebaseInstanceIDService";
 
     @Override
     public void onTokenRefresh() {
-
-        String refreshedToken = FirebaseInstanceId.getInstance().getToken();
-        Log.d(TAG, refreshedToken);
+        super.onTokenRefresh();
+        String token = FirebaseInstanceId.getInstance().getToken();
+        Log.d(TAG, token);
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/data/source/local/MyFirebaseInstanceIDService.java:19]: ERROR: [LongLogTag] The logging tag can be at most 23 characters, was 27 (MyFirebaseInstanceIDService)
 Log tags are only allowed to be at most 23 tag characters long.

Comment (tuanlvt-1117) [reply app/src/main/java/com/framgia/forder/data/source/local/MyFirebaseInstanceIDService.java:19]: Log Token

**Pull request merged**